docs/index.md

```diff
@@ -371,6 +371,88 @@ print(Settings().model_dump())
 #> {'numbers': [1, 2, 3]}
 ```
 
+### Disabling JSON parsing
+
+pydatnic-settings by default parses complex types from environment variables as JSON strings. If you want to disable
+this behavior for a field and parse the value by your own, you can annotate the field with `NoDecode`:
```

Comment:
```suggestion
this behavior for a field and parse the value in your own validator, you can annotate the field with `NoDecode`:
```
Will `NoDecode` and `ForceDecode` be included in API documentation? They should, if so we should link to the API documentation here.